include/framework/everest.hpp

```diff
@@ -137,30 +140,17 @@ class Everest {
 
     ///
     /// \returns the instance of the Everest singleton taking a \p module_id, the \p config, a \p mqtt_server_address
-    /// and \p mqtt_server_port as parameters. If validation of data with the known json schemas is needed this can be
-    /// activated by setting \p validate_data_with_schema to true
+    /// , \p mqtt_server_port , \p mqtt_everest_prefix and \p mqtt_external_prefix as parameters. If validation of data
+    /// with the known json schemas is needed this can be activated by setting \p validate_data_with_schema to true
     static Everest& get_instance(std::string module_id, Config config, bool validate_data_with_schema,
```

Comment:
this got quite large, maybe all the parameters should go into a meaningful struct?, actually most of the parameters could be part of the Config parameter

Reply:
Absolutely, I would like to do some general restructuring here at some point, there's quite a lot of code duplication in various parts  concerning these parameters. Especially in the JavaScript and Python binding code